compiler/wasm/src/compile.rs

```diff
@@ -17,6 +17,9 @@ use crate::errors::JsCompileError;
 
 const BACKEND_IDENTIFIER: &str = "acvm-backend-barretenberg";
 
+const NOIR_ARTIFACT_VERSION_STRING: &str =
+    concat!(env!("CARGO_PKG_VERSION"), "-", env!("GIT_COMMIT"));
+
```

Comment:
These two being here seems a bit odd -- I think versioning could probably go into the driver/nargo and then have it be shared between nargo_cli and wasm